tests/arch/arm/arm_sw_vector_relay/src/arm_sw_vector_relay.c

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Just curious: Why a separate file?

Reply:
Other than main.c? I guess that's how the rest of the arch/arm test suite is designed, just followed the style